### ISSUE TITLE
Windows no longer boots through opencore

### ISSUE THREAD

**Issue by ESProcessing** [EETagent/T480-OpenCore-Hackintosh#71]
After the latest round of ACPI and config.plist consolidations booting OSX works fine as expected, but booting into windows fails.

**Comment by EETagent** [comment]
True, that's one of the reasons I did not create new release yet. I'm sorry that I forgot to update that in Readme.

Currently best way is to chain OpenCore with rEFInd or use boot menu entries.

**Comment by EETagent** [comment]
https://github.com/EETagent/T480-OpenCore-Hackintosh/issues/15#issuecomment-678799038

Last time, it was all about Return(Zero)

**Comment by skolodzieski** [comment]
Thanks. I have based my t470 build upon a merge or your repository with mine. It all went very easily. I will have a better look at the SSDT's. I think I only introduced 1 additional one that enables additional function keys using ThinkpadAssistant. Thanks for all the work. I like the last round of patches, the consolidated battery patch is cool.

**Comment by ESProcessing** [comment]
That previous comment was by me. Sometime I forget to switch from my work sccount to my personal account :)

**Comment by kai2ou** [comment]
> Fixed supplying non-RT SetVirtualAddressMap for non-macOS systems
the 0.6.6

**Comment by snazzybunny** [comment]
Does booting into Windows still work for you guys? I am getting the circular loading screen then reboot to lenovo screen. Thanks.

**Comment by aleixsr** [comment]
No, it is not working for me currently.
